CMakeLists.txt

```diff
@@ -52,6 +52,9 @@ option(SKIP_PACKAGING "" OFF)
 # be verbose about flags used
 option(VERBOSE "be verbose about flags used" OFF)
 
+# use standalone boost asio
+add_definitions("-DARANGODB_STANDALONE_ASIO=1")
```

Comment:
In 3.3 sollte hier der default 0 sein, damit man beim Compilieren von außen nichts merkt.

Reply:
This is indeed the case in 3.3